src/drivers/SC7A20.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include "AccelerationSensor.h"
+
+namespace Pinetime {
+  namespace Drivers {
+    class SC7A20: public AccelerationSensor {
+    public:
+      SC7A20(TwiMaster& twiMaster, uint8_t twiAddress);
+      SC7A20(const SC7A20&) = delete;
+      SC7A20& operator=(const SC7A20&) = delete;
+      SC7A20(SC7A20&&) = delete;
+      SC7A20& operator=(SC7A20&&) = delete;
+
+      /// The chip freezes the TWI bus after the softreset operation. Softreset is separated from the
+      /// Init() method to allow the caller to uninit and then reinit the TWI device after the softreset.
+      void SoftReset();
+      void Init();
+      AccelerationValues Process();
+      void ResetStepCounter();
+      void SetMotion(MotionEvents event);
+      MotionEvents GetMotionInfo();
+
+    private:
+      int16_t average(int16_t *items, uint8_t len);
+      void filter(int16_t *x, int16_t *y, int16_t *z);
+
+      bool isResetOk = false;
+    };
+  }
+}
```

Comment:
please add a newline at the end of the file

```suggestion
}

```